src/PrestaShopBundle/Entity/Repository/ApiAccessRepository.php

```diff
@@ -54,4 +55,12 @@ public function deleteByApplication(AuthorizedApplication $application): void
 
         $query->execute();
     }
+
+    public function save(ApiAccessInterface $apiAccess): int
```

Comment:
Please use the Doctrine entity here for two reasons:
- the doctrine entity manager is unable to persist an object that simply implements the interface but is not a doctrine entity
- the principle of interfaces ADR has been refused an this interface should be clean soon